src/LibXmlValidator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\XmlValidator;
+
+use DOMDocument;
+use function array_map;
+use function count;
+use function libxml_clear_errors;
+use function libxml_get_errors;
+use function libxml_use_internal_errors;
+
+trait LibXmlValidator
```

Comment:
being a trait to compose rather than a usable class, may be called with a different name like `LibXmlWrapper` or similar

Reply:
Wasn't quite sure what to call it. Not sure `wrapper` works. Let's keep it for now.